src/Client/ClientBaseHelpers.cpp

```diff
@@ -237,4 +238,25 @@ void highlight(const String & query, std::vector<replxx::Replxx::Color> & colors
 }
 #endif
 
+String formatQuery(String query)
+{
+    bool multiline_query = query.contains('\n');
+    ParserQuery parser(query.data() + query.size(), /*allow_settings_after_format_in_insert_=*/ false, /*implicit_select_=*/ false);
+    const ASTPtr ast = parseQuery(parser, query, /*max_query_size=*/ 0, /*max_parser_depth=*/ 0, /*max_parser_backtracks=*/ 0);
```

Comment:
This will probably throw an exception on multi-statement query. Let's use `parseQueryAndMovePosition` and format each of the parsed queries independently?

```
:) select 1; select 2; select 10;
Syntax error (. Multi-statements are not allowed): failed at position 9 (end of query): ; select 2; select 10;.
```

Reply:
Good catch, fixed